src/test/java/spaceEngineers/InteractionTest.kt

```diff
@@ -0,0 +1,56 @@
+package spaceEngineers
+
+import org.junit.jupiter.api.Assertions.assertNotNull
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import spaceEngineers.SeRequest
+import spaceEngineers.commands.*
+import testhelp.checkMockObservation
+import testhelp.environment
+
+class InteractionTest {
+    @Test
+    fun equipToolbarItemTest() = environment {
+        val observation = getSeResponse(SeRequest.command(
+                SeAgentCommand.interact("you", InteractionArgs(InteractionType.EQUIP, 4))))
```

Comment:
a param disappeared from IntegrationArgs args, there shoudl be allowSizeChange -> true